tests/test_interventions.py

```diff
@@ -114,11 +114,51 @@ def test_change_people_state():
     return s0, s1, s2
 
 
+def test_add_method():
+    """ Test the add_method intervention """
+    sc.heading('Testing add_method()...')
+    
+    pars = dict(n_agents=500, start=2000, stop=2015, verbose=0, location='kenya')
+
+    # Test 1: Basic add_method functionality
+    method_pars = dict(
+        name='test_method',
+        label='Test Method',
+        efficacy=0.999,
+        dur_use=ss.lognorm_ex(ss.years(2), ss.years(0.5)),  # TODO - should not need this, but fails without it!
```

Comment:
@menriquez-IDM could you help with this? I'm so perplexed! It fails if I delete this line, but I was hoping it would automatically copy...